app/data/census_household.json

```diff
@@ -893,15 +910,16 @@
                 {
                     "type": "interstitial",
                     "id": "household-member-begin",
-                    "title": "{{ answers.person_full_name[group_instance] }}",
+                    "title": "{{ [answers.first_name[group_instance]|default('', true), answers.last_name[group_instance]|default('', true)]|join(' ') }}",
```

Comment:
@ajmaddaford made comment about not having the names in the block titles.
https://github.com/ONSdigital/eq-survey-runner/pull/598#pullrequestreview-11560954

Not sure if its worth ripping them out here?